packages/application-components/src/theming.ts

```diff
@@ -0,0 +1,25 @@
+import kebabCase from 'lodash/kebabCase';
+import { designTokens } from '@commercetools-uikit/design-system';
+
+export const themesOverrides = {
+  default: {
+    paddingVerticalForPage: designTokens.spacingM,
+    paddingHorizontalForPage: designTokens.spacingM,
+  },
+  test: {
+    paddingVerticalForPage: designTokens.spacingXl,
+    paddingHorizontalForPage: '40px',
+  },
+};
+
+export const appKitDesignTokens = Object.values(themesOverrides).reduce(
+  (appKitTokens, themeTokens) => {
+    Object.keys(themeTokens).forEach((tokenName) => {
+      if (!appKitTokens[tokenName]) {
+        appKitTokens[tokenName] = `var(--${kebabCase(tokenName)})`;
```

Comment:
Since this are new tokens I don't think we need default values for it, but it depends on the decision we need to make about if we will support using `app-kit` components without also using a `ThemeProvider`.